repo/plugin.video.nzbdav/resources/lib/resolver_submit.py

```
@@ -0,0 +1,637 @@
+    thread = _resolver.threading.Thread(target=target, name=thread_name, daemon=True)
+    try:
+        thread.start()
+    except RuntimeError:
+        target()
+
+
+def _safe_probe_by_name(find_fn, title, settings_getter, probe_label):
+    """Call a find-by-name probe, logging+swallowing any error (returns None)."""
+    try:
+        return find_fn(title, **_resolver._settings_getter_kwargs(settings_getter))
+    except Exception as e:  # pylint: disable=broad-except
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)